src/helpers/matchers/accessibilityValue.ts

```diff
@@ -0,0 +1,24 @@
+import { AccessibilityValue } from 'react-native';
+import { ReactTestInstance } from 'react-test-renderer';
+import { TextMatch } from '../../matches';
+import { matchStringProp } from './matchStringProp';
+
+export interface AccessibilityValueMatcher {
+  min?: number;
+  max?: number;
+  now?: number;
+  text?: TextMatch;
+}
+
+export function matchAccessibilityValue(
+  node: ReactTestInstance,
+  matcher: AccessibilityValueMatcher
+): boolean {
+  const value = (node.props.accessibilityValue ?? {}) as AccessibilityValue;
```

Comment:
Does `const value: AccessibilityValue = node.props.accessibilityValue ?? {}` typecheck? That's usually safer, `as` can lead to bugs

Reply:
`node.props.accessibilityValue` is `any` as `ReactTestInstances` props are not typed.

Reply:
Then it should still pass, so I'd suggest to use this pattern, it's better to have this habit :)

Reply:
done